audiorecorder/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1 @@
+application=org.odk.collect.audiorecorder.TestApplication
```

Comment:
This was the magic I needed to get Dagger setup properly for tests. I'd completely forgotten robolectric could be configured in a properties file.